### PR TITLE
🎨 Add missing typescript to project

### PR DESCRIPTION
**Pull request by fre-ben**
Updated the project with types.
In the process of writing the functions I got carried away and forgot to define types.

### CODE REVIEW

**Review by RonjaMeinecke — COMMENTED** [review]
💥💥💥LGTM💥💥💥

**Review by RonjaMeinecke — APPROVED** [review]

File: components/gameboard/TotalScore.tsx [hunk 1]
```diff
@@ -16,7 +17,7 @@ function TotalScore() {
     return (
       <li key={player.name} className={styles.playerListItem}>
         <span className={styles.playerName}>{player.name}</span>{" "}
```
Comment (mewiemers) [inline components/gameboard/TotalScore.tsx:19]:
```suggestion
        <span className={styles.playerName}>{player.name}</span>{" "}
```
Do you need this? {" "}

File: components/gameboard/TotalScore.tsx [hunk 2]
```diff
@@ -16,7 +17,7 @@ function TotalScore() {
...
-        <span className={styles.playerScore}>{player.score}</span>
+        <span className={styles.playerScore}>{player.totalScore}</span>
```
Comment (mewiemers) [inline components/gameboard/TotalScore.tsx:20]: can you better destructure here?

Comment (fre-ben) [reply components/gameboard/TotalScore.tsx:20]: Yup, will do this

File: components/gameboard/TotalScore.tsx (hunk 1 above)
Comment (fre-ben) [reply components/gameboard/TotalScore.tsx:19]: I think I need it for the whitespace between the spans, but I'll check again. Thanks :)

**Pull request merged**